lib/global-styles.php

```diff
@@ -245,7 +245,7 @@ function gutenberg_global_styles_filter_post( $content ) {
 		$data_to_encode = WP_Theme_JSON_Gutenberg::remove_insecure_properties( $decoded_data );
 
 		$data_to_encode['isGlobalStylesUserThemeJSON'] = true;
-		return wp_json_encode( $data_to_encode );
+		return wp_json_encode( addslashes( $data_to_encode ) );
```

Comment:
In adding and removing slashes, we should prefer `wp_slash` and `wp_unslash` as they've been optimized to run consistently in the variety of PHP environments (slashes are a deeply weird area of PHP).

Reply:
Oh, now reading this again, we need to reverse the order as well. Slashing needs to happen at the end, not the start.

```php
return wp_slash( wp_json_encode( $data_to_encode ) );
```

Reply:
Updated